package/yast2-network.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Thu May 25 09:27:09 UTC 2017 - mfilka@suse.com
+
+- bnc#1039532
+  - do not write hostname into /etc/hosts duplicitly for one IP
+  - configure static IPs to resolve to system wide hostname during
+    installation
```

Comment:
ah I finally see ... I forgot to write half of sentence as I already wrote it 2 or 3 times ;-)